pkg/test/framework/components/istio/operator.go

```diff
@@ -124,17 +137,46 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 	i.id = ctx.TrackResource(i)
 
 	if !cfg.DeployIstio {
-		scopes.Framework.Info("skipping deployment due to Config")
+		scopes.Framework.Info("skipping deployment as specified in the config")
 		return i, nil
 	}
 
-	istioCtl, err := istioctl.New(ctx, istioctl.Config{})
+	// Top-level work dir for Istio deployment.
+	workDir, err := ctx.CreateTmpDirectory("istio-deployment")
 	if err != nil {
 		return nil, err
 	}
 
-	// Top-level work dir for Istio deployment.
-	workDir, err := ctx.CreateTmpDirectory("istio-deployment")
+	// Create and push the CA certs to all clusters to establish a shared root of trust.
+	root, err := newSelfSignedRootCA(workDir)
```

Comment:
Should we only do this in multi cluster mode? Is there any pros/cons to doing this if we only have one cluster?

Reply:
Sure ... done.